qsiprep/niworkflows/interfaces/segmentation.py

```diff
@@ -15,49 +15,6 @@
 from . import report_base as nrc
 from .. import NIWORKFLOWS_LOG
 
-
```

Comment:
This piece of code isn't under the FSL License, so we can keep it here

Reply:
added back